### DISCUSSION TITLE
Restricted Slack invite links

### DISCUSSION THREAD

**Discussion by eldorel** [bitfocus/companion#2555] [Q&A] [answered]
I'm not sure if this was intentional, but the current slack invites are restricted to users with bitfocus/internal email addresses.   

> You can use any account with the domain:
>
> bitfocus.io
> soniccity.no
>
> Don’t have an email address from one of those domains?
> Contact the workspace administrator at Bitfocus AS for an invitation.

Since that link is all over the UI/github/website, I assume it was meant to be a public space.

**Comment by Julusian** [comment, marked as answer]
The invite links expire after a certain number of uses. 

https://join.slack.com/t/bitfocusio/shared_invite/zt-221zt00sw-9QaViFkbQcgCYFlX_MU~vg will work, Ill get that other link updated

**Reply by eldorel** [reply]
Thank you very much.

**Comment by mihindupaul** [comment]
@Julusian we also looking for invitation link to register a new module. can you please share one

**Reply by Julusian** [reply]
https://join.slack.com/t/bitfocusio/shared_invite/zt-2h4vetyef-xWVNpJ4hIsEsSlyNJMguXQ

**Comment by Gfisk25** [comment]
Hi,
I'd like to start developing modules.  Can you please post a new link.
Thanks!

**Reply by Julusian** [reply]
The invite link has been updated https://bitfocus.io/api/slackinvite

**Comment by saamerm** [comment]
@Julusian Seems like it's expired again!

**Reply by Julusian** [reply]
@saamerm Updated

**Comment by Takabrycheri** [comment]
Even https://bitfocus.io/api/slackinvite is locked to bitfocus.io and soniccity.no

**Reply by Julusian** [reply]
The link expired again and has been updated

**Comment by greatmastix** [comment]
I think it might have expired again

**Reply by dorianim** [reply]
Yep, same issue here

**Reply by Julusian** [reply]
should be fixed now. It's an annoying manual process without an easy way to monitor it for expiry

**Reply by saamerm** [reply]
@Julusian thanks for doing this! Is there perhaps a reminder bot that might help ?

**Reply by Julusian** [reply]
It expires after 400 uses, not a time period. So it needs to be something with active monitoring or perhaps watching the user count growth. 
I havent tried asking copilot/claude to come up with something though, last looked at this before they were good enough to do much

**Reply by saamerm** [reply]
haha makes sense, you could also do it to where people click on a link that increases the count and then redirects. Once it reaches the 400 count, it notifies you. Won't be accurate, but its something
